db/people.yml

```diff
@@ -2116,24 +2116,30 @@ jchung:
   position: unknown
 jeaster:
   active: true
-  aka: []
+  aka: Jay Easter
   avatar: https://github.com/easterjaylah.png
-  bio: ''
-  education: []
+  bio: 'Jay Easter is a Henson Entrepreneur Endowed Honors scholar at the
+University of Maryland Eastern Shore, studying general engineering with a
+mechanical engineering specialization. His general research interests pertain to
+understanding the relationship between the atomic structure and the emergent
+properties of organic materials for applications in energy, and the numerical modelling 
+and fabrication of NEMS devices. Jay is passionate about entrepreneurship and the 
+integration of historically underrepresented groups in STEM.'
+  education: BS General Engineering, mech. specilization (Expected Graduation: 5/2023)
```

Comment:
this should be a dict, not a string (ask Jaylyn or one of the others what this means).  I recommend to read a bit about Python data types to get some background, and also about the yaml file format, so you are more comfortable here.